### PR TITLE
feat: cc_import for libpython

### PR DESCRIPTION
**Pull request by f0rmiga**
Exposes a `cc_import` target for libpython. So far, we've only been exposing the headers but on hermetic environments without libpython being found in the system search paths, we need to expose those libraries too.

### CODE REVIEW

File: python/repositories.bzl [hunk 1]
```diff
@@ -148,6 +148,16 @@ cc_library(
     ],
 )
 
+cc_import(
+    name = "libpython",
+    hdrs = [":includes"],
+    shared_library = glob([
```
Comment (UebelAndre) [inline python/repositories.bzl:154]: Can you make this a select so it can be more explicit? Shouldn't all toolchains be consistent in exposing this?

Comment (f0rmiga) [reply python/repositories.bzl]: Fixed in 043f2e388dcf6854260253ef19e7079d32babbbe.

Comment (f0rmiga) [reply python/repositories.bzl]: Windows is always the weird one.

**Review by hrfuller — APPROVED** [review]
LGTM.

**Pull request merged**